### PR TITLE
feat: add events to Profile

### PR DESCRIPTION
**Pull request by zimpha**

### CODE REVIEW

File: src/interfaces/IProfile.sol
Comment (Thegaram) [inline src/interfaces/IProfile.sol:1]: Ok, but will it not be a problem that the events are missing for some historical actions?

File: src/profile/Profile.sol
Comment (Thegaram) [inline src/profile/Profile.sol:1]: Will we upgrade to this impl on Sepolia once this PR is merged?

**Review by Thegaram — APPROVED** [review]

File: src/interfaces/IProfile.sol
Comment (zimpha) [reply src/interfaces/IProfile.sol:1]: not a big problem, offchain indexer can fetch onchain state as initial state based on the upgrade block number.

File: src/profile/Profile.sol
Comment (zimpha) [reply src/profile/Profile.sol:1]: yes

**Pull request merged**